src/special_elixirs.jl

```diff
@@ -0,0 +1,170 @@
+
+# Apply the function `f` to `expr` and all sub-expressions recursively.
```

Comment:
Just from a user convenience point of view: Can we maybe move the auxiliary methods to the bottom (`walkexpr`, `replace_assignments` and `find_assignments`)? That way, the important stuff comes first, the scaffolding at the bottom.

Reply:
Do you mean after `trixi_include` or at the very bottom of the file?

Reply:
I assume the latter - done in a8da157.

Reply:
:+1: